### ISSUE TITLE
Control Login/Registration page markup to meet accessibility guidelines

### ISSUE THREAD

**Issue by CrispinF** [i7MEDIA/mojoportal#109]
An accessibility test on a mojoPortal site requires that the Login and Registration pages have no heading and so "Login" and "Create a New Account" texts should be `<h1>`.  This is following the modern school of thought that each page should have a unique H1, rather than using that for the site name.
I've reviewed https://www.mojoportal.com/login-module-display-settings and can't see how we can influence this without altering core code. Is it possible?
Changing this would also presumably mean losing the fieldset, as having both h1 and fieldset legend would be odd duplication, unless you can come up with complementary texts that work in all scenarios.

**Comment by JosephMDavis** [comment]
Hi Crispin,

I'm not sure understand what you mean by the accessibility test requires that these pages not have a heading and then asking that the legends be turned into h1 elements. 

We're good with allowing control over this though so we've written up a new CoreDisplaySettings control, to be used in the layout.master. Defaults are shown here
```
<portal:CoreDisplaySettings runat="server" 
  DefaultPageHeaderMarkupTop="<h1>"
  DefaultPageHeaderMarkupBottom="</h1>"
/>
```

We've removed the fieldset and legend. There will not be a configurable way to add those back, either.

We'll be adding some other things to this control in the near future.

**Comment by CrispinF** [comment]
@JosephMDavis sorry that was my typo. I meant the test requires that they DO have a heading! Your changes look sensible, thanks.
